install_config/install/advanced_install.adoc

```diff
@@ -1690,6 +1823,87 @@ ifdef::openshift-origin[]
 endif::[]
 ----
 
+[discrete]
+[[running-the-advanced-installation-tsb]]
+==== Optional: Deploying the Template Service Broker
+
+If you have xref:enabling-service-catalog[enabled the service catalog] and want
+to deploy the xref:configuring-template-service-broker[template service broker],
+run the following manual steps after the cluster installation completes
+successfully:
+
+[WARNING]
+====
+The template service broker (TSB) is currently a Technology Preview feature and
+should not be used in production clusters. Enabling the TSB currently requires
+opening unauthenticated access to the cluster; this security issue will be
+resolved before exiting the Technology Preview phase.
+====
+
+. Ensure that one or more source project for the TSB were defined via
+`openshift_template_service_broker_namespaces` as described in
+xref:../../install_config/install/advanced_install.html#configuring-template-service-broker[Configuring the Template Service Broker].
+
+. Run the following command to enable unauthenticated access for the TSB:
++
+----
+$ oc adm policy add-cluster-role-to-group \
+    system:openshift:templateservicebroker-client \
+    system:unauthenticated system:authenticated
+----
+
+. Create a *_template-broker.yml_* file with the following contents:
++
+[source,yaml]
+----
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: Broker
+metadata:
+  name: template-broker
+spec:
+  url: https://kubernetes.default.svc:443/brokers/template.openshift.io
+----
+
+. Use the file to register the broker:
++
+----
+$ oc create -f template-broker.yml
+----
+
+. Enable the Technology Preview feature in the web console to use the TSB instead
+of the standard `openshift` global library behavior.
```

Comment:
yes.  Essentially it disables the old behavior of the console which was to explicitly look for templates in the openshift namespace.  Setting this tells the web console not to do that, and to instead expect that the templates will appear as part of the service catalog along w/ other service offerings.